src/nextcloud/scripts/nextcloud-cron

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+export NEXTCLOUD_CONFIG_DIR=$SNAP_DATA/nextcloud/config
```

Comment:
The reason you ran into issues at first is because upon first install it takes a little while for this directory to exist. In order to avoid what looks like an error (but isn't), I suggest before starting the cron loop below, you sit and wait for this directory to exist. Something like the following:

echo -n "Waiting for Nextcloud config dir... "
while [ ! -d "$NEXTCLOUD_CONFIG_DIR" ]; then
    sleep 1
done
echo "done"